charts/stable/rtsp-to-web/values.yaml

```diff
@@ -9,11 +9,11 @@ image:
   # -- image repository
   repository: ghcr.io/deepch/rtsptoweb
   # -- image tag
-  tag: v1.0.0
+  tag: v2.2.0
```

Comment:
```suggestion
  tag:
```

Reply:
This will default to the `appVersion` in the `Chart.yaml`. Allows us to only maintain the container version in one place.